### ISSUE TITLE
Share your PersonalHomeKit :D

### ISSUE THREAD

**Issue by arnauddorgans** [etwmc/Personal-HomeKit-HAP-#24]
Hey, 
i made a little video for you guys, this is my installation using PersonalHomeKit :p 
https://www.youtube.com/watch?v=kyxYWq47lxA

**Comment by PSForan** [comment]
@Arnoymous do you have a blog? Interested in learning about your setup...how your php interfaces with the rpi & then, well, everything else!

**Comment by prsolucoes** [comment]
Anyone knows about make it work with arduino?

**Comment by beckmx** [comment]
it doesnt work with arduino, because it needs a faster CPU, the iPhone
would give u a timeout just because of the time

2017-01-05 0:14 GMT-06:00 prsolucoes <notifications@github.com>:

**Comment by prsolucoes** [comment]
Hi,

And on a ESP 8266, 160mhz?

We can do it on ESP?

**Comment by beckmx** [comment]
yes, I tried but there is no documentation on how to compile with a
certified SDK, so there are not SSL libs valid, so, you might try and post
ur results here

2017-01-05 11:37 GMT-06:00 prsolucoes <notifications@github.com>:

**Comment by prsolucoes** [comment]
Hi,

Do you have any chat to talk? Like hangout or skype?

**Comment by beckmx** [comment]
I do but I don't have any code for what u want to do

2017-01-05 12:32 GMT-06:00 prsolucoes <notifications@github.com>:

**Comment by prsolucoes** [comment]
No problem, only a better contact to exchange idea.

**Comment by etwmc** [comment]
I think there is crypto library (specially, just SHA512) on ESP8266? I saw it in the past but forget where I found, or if it works at all. 
I mean, I included all security functions beside SHA512, because in some case there is hardware accelerator/platform optimized codes which will definitely better than the code I would ship (a.k.a. textbook implementation on SHA512)

I'm busy on job hunting now so right now can't actually building it. (Well, mainly cash-strapped. Didn't have a dev board sitting by. )
